src/window/building/common.c

```diff
@@ -53,24 +53,34 @@ int window_building_get_vertical_offset(building_info_context *c, int new_window
     return new_window_y;
 }
 
-void window_building_draw_employment(building_info_context *c, int y_offset)
+void window_building_draw_employment(building_info_context* c, int y_offset)
 {
-    building *b = building_get(c->building_id);
+    do_window_building_draw_employment(c, y_offset, 1);
+}
+
+void do_window_building_draw_employment(building_info_context* c, int y_offset, int consider_house_covering)
+{
+    building* b = building_get(c->building_id);
     int text_id;
     if (b->num_workers >= model_get_building(b->type)->laborers) {
         text_id = 0;
-    } else if (city_population() <= 0) {
+    }
+    else if (city_population() <= 0) {
```

Comment:
Please keep to the project's coding/spacing conventions: `}` and the corresponding `else` on one line.

Reply:
Done too. In fact theses modifications happen when i copy/paste. I don't use Visual studio often. I will check if there is a parameter to automatically change the coding style, or a file to it.